static/templates/settings/admin_bot_form.hbs

```diff
@@ -13,6 +13,14 @@
             <label for="user_id">{{t "User ID" }}</label>
             <input type="text" autocomplete="off" name="user_id" value="{{ user_id }}" readonly/>
         </div>
+        <div class="input-group">
+            <label class="input-label" for="bot-role-select">{{t 'Role' }}
+                {{> ../help_link_widget link="/help/roles-and-permissions" }}
```

Comment:
@alya @laurynmm should this link to the roles and permissions page in the API documentation instead? We can then potentially enhance that page with a new section about giving bot users roles.

Reply:
What's the page? I'm not familiar with API documentation at all, and I don't see it in the left sidebar.

Reply:
@alya - We added that page recently and it looks like the general zulip docs haven't been updated for that yet. You can see it on the [CZO API docs](https://chat.zulip.org/api/roles-and-permissions).

I feel like adding information to the roles and permissions API page about bot users having roles could be useful, but in a context of how clients would use/need that information.

For this, I wonder if we want to update (and link to) the [about bots](https://chat.zulip.org/help/bots-and-integrations) page to be clearer about bot users roles, and link to the help center's roles and permissions doc from there?

Reply:
As discussed on our call, that make sense to me.

Reply:
@laurynmm can you make sure this gets captured in an issue or your TODO list? I'm planning to merge this PR.

Reply:
:+1:  I've got it on my to-do for now